lizrd/grid/setup_arguments.py

```diff
@@ -0,0 +1,49 @@
+import os
+from typing import List, Optional
+
+
+def make_singularity_mount_paths(setup_args: dict, training_args: dict) -> str:
```

Comment:
Może wrzucić to wewnątrz klasy MachineBackend jako statyczną metodę, bo, tylko tam będzie używane + zmienić, żeby przyjmowało hf_datasets_cache, train_dataset_path i validation_dataset_path zamiast słowników